python/kfserving/setup.py

```diff
@@ -12,6 +12,11 @@
     packages=find_packages("kfserving"),
     install_requires=[
         "tornado >= 1.4.1",
-        "argparse >= 1.4.0"
+        "argparse >= 1.4.0",
+        "tensorflow"
```

Comment:
>tensorflow [](start = 9, length = 10)

ugh, seems like a heavyweight dependency to tack onto the base

Reply:
I agree. The Seldon protocol just needs the TensorProto compiled protos and the function `tf.make_ndarray` ([defn](https://www.tensorflow.org/api_docs/python/tf/make_ndarray)).  Also, when we support grpc I think we will need tensorflow to handle the messages being sent.

Reply:
This will be compounded if we decide to use KFServer to serve Explainer/PayloadLogger/OutlierDetector components as well.

Reply:
Removed dependency for now.